Formula/feedgnuplot.rb

```diff
@@ -1,8 +1,8 @@
 class Feedgnuplot < Formula
   desc "Tool to plot realtime and stored data from the command-line"
   homepage "https://github.com/dkogan/feedgnuplot"
-  url "https://github.com/dkogan/feedgnuplot/archive/v1.55.tar.gz"
-  sha256 "1205afedf8ce79d8531e0d0f8f9565df365a568a0ee6a8e17738602682095303"
+  url "https://github.com/dkogan/feedgnuplot/archive/v1.56.tar.gz"
+  sha256 "93943aee58f124cc21f70267fcfba48ec70a8dc112ddcba075afe074b51270bc"
   # licensed under either "GPL-3.0" or "Artistic-1.0"
   license "GPL-3.0"
```

Comment:
[Upstream feedgnuplot.spec](https://github.com/dkogan/feedgnuplot/blob/c8dac15db3b409c9a69b11b017dae2d838d3e4da/feedgnuplot.spec) states license as `Artistic or GPL-1+`.
```suggestion
  license "GPL-1.0-or-later"
```

Reply:
We can update using the new `any_of:` license syntax.

Reply:
We can also use `any_of` so that a comment would be unnecessary.